src/countdown/Countdown.tsx

```diff
@@ -0,0 +1,72 @@
+import "./_countdown.scss";
+
+import React, {useLayoutEffect} from "react";
+import classNames from "classnames";
+
+import useCountDownTimer from "../core/utils/hooks/useCountdownTimer";
+import List from "../list/List";
+import ListItem from "../list/item/ListItem";
+
+export interface CountdownProps {
+  testid: string;
+  startDate: Date;
+  countDownIntervalInSeconds?: number;
+  onEnd?: () => void;
+  customClassName?: string;
+}
+
+function Countdown({
+  testid,
+  startDate,
+  onEnd,
+  countDownIntervalInSeconds = 1,
+  customClassName
+}: CountdownProps) {
+  const countDownData = useCountDownTimer(startDate, countDownIntervalInSeconds);
+  let items: {id: "days" | "hours" | "minutes" | "seconds"; count: string | number}[];
+
+  if (countDownData.days >= 1) {
+    items = [
+      {id: "days", count: countDownData.days},
+      {id: "hours", count: countDownData.hours},
+      {id: "minutes", count: countDownData.minutes}
+    ];
+  } else if (countDownData.hours >= 1) {
+    items = [
+      {id: "hours", count: countDownData.hours},
+      {id: "minutes", count: countDownData.minutes}
+    ];
```

Comment:
What if I want to see seconds even though `countDownData.hours >=1` or `countDownData.days >= 1`?  Maybe we can add a prop to `Countdown` as `alwaysShowSeconds`. This can be optional and default to `false`. 

```suggestion
  } else if (countDownData.hours >= 1) {
    items = [
      {id: "hours", count: countDownData.hours},
      {id: "minutes", count: countDownData.minutes},
      alwaysShowSeconds && {id: "seconds", count: countDownData.seconds}
    ].filter(Boolean) as CountDownItem[];
```